root/meta/naming/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 ROOTTEST_ADD_AUTOMACROS(DEPENDS namingMatches.cxx
-                        EXCLUDE execCheckNaming)
+                        EXCLUDE execCheckNaming execClassEditNormalize)
 
 ROOTTEST_ADD_TEST(execCheckNaming
                   MACRO execCheckNaming.C
                   OUTREF execCheckNaming${ref_suffix})
+
+if(NOT MSVC OR CMAKE_CXX_STANDARD LESS 17)
```

Comment:
Are those removal(s) 'temporary'?

Reply:
@pcanal  Nope. As mentioned in the commit log, the `std::unary_function` and `std::binary_function` functions have been deprecated in C++11, and removed in C++17. Microsoft removed them, and the other compilers didn't, but it's only a matter of time...

Reply:
Thanks.  In this, I think the std::*_function are not functionally used and could be replaced by inactive equivalent struct (as the structure of the user code is being probed rather than its functionality).